### PR TITLE
all tests are done

### PR DESCRIPTION
**Pull request by chezi-nikop**
all test are done

### CODE REVIEW

File: src/test/kotlin/com/hibob/kotlinBasic/BankAcccuntTest.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+        @Test
+        fun `withdraw amount greater than balance throws IllegalArgumentException`() {
+            assertThrows<IllegalArgumentException> {
+                BankAccount(100.0).withdraw(101.0)
+            }
+        }
+
+           @Test
+           fun `withdraw negative or zero amount throws IllegalArgumentException`() {
```
Comment (kerensegev) [inline src/test/kotlin/com/hibob/kotlinBasic/BankAcccuntTest.kt:42]: The code should be aligned  ( I mean you have unnecessary tabs here)

File: src/test/kotlin/com/hibob/kotlinBasic/ListManagerTest.kt [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+ * Test with one person.
+ * Test with multiple people to ensure they are sorted first by age, then by name.
+ * Test with edge cases like people with the same name but different ages and vice versa.
+ *
+ */
+class ListManagerTest {
+
+    @Test
+    fun `Test adding a unique person`() {
+        assertEquals(Person("chezi nikop", 27), Person("chezi nikop", 27))
```
Comment (kerensegev) [inline src/test/kotlin/com/hibob/kotlinBasic/ListManagerTest.kt:35]: You didn't add the person to the list

File: src/test/kotlin/com/hibob/kotlinBasic/ListManagerTest.kt [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+        assertEquals(2, listManager.sumOfPeople())
+    }
+
+    @Test
+    fun `removing a person that exists in the list`() {
+        val listManager = ListManager()
+        listManager.addPerson(person = Person("chezi nikop", 27))
+
+        assertEquals(true, listManager.removePerson(person = Person("chezi nikop", 27)))
```
Comment (kerensegev) [inline src/test/kotlin/com/hibob/kotlinBasic/ListManagerTest.kt:62]: You also need to make sure it was actually removed

File: src/test/kotlin/com/hibob/kotlinBasic/ListManagerTest.kt [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+    @Test
+    fun `Test the state of the list after multiple add and remove operations`() {
+        val listManager = ListManager()
+        listManager.addPerson(person = Person("chezi nikop", 27))
+        listManager.addPerson(person = Person("eli cohen", 27))
+        listManager.removePerson(person = Person("chezi nikop", 27))
+
+        assertTrue(listManager.containsPeople(Person("eli cohen", 27)))
+        assertEquals(false, listManager.containsPeople(Person("chezi nikop", 27)))
```
Comment (kerensegev) [inline src/test/kotlin/com/hibob/kotlinBasic/ListManagerTest.kt:79]: You can use assertFalse

**Review by kerensegev — APPROVED** [review]

**Review by kerensegev — APPROVED** [review]
